packages/create-astro/src/actions/typescript.ts

```diff
@@ -90,11 +90,18 @@ const FILES_TO_UPDATE = {
 	) => {
 		try {
 			// add required dependencies for astro check
-			if (options.ctx.install)
-				await shell(options.ctx.packageManager, ['add', '@astrojs/check', 'typescript'], {
-					cwd: path.dirname(file),
-					stdio: 'ignore',
-				});
+			let installSucceeded = true;
+			if (options.ctx.install) {
+				const { exitCode } = await shell(
+					options.ctx.packageManager,
+					['add', '@astrojs/check', 'typescript'],
+					{
+						cwd: path.dirname(file),
+						stdio: 'ignore',
+					}
+				);
+				installSucceeded = exitCode === 0;
+			}
```

Comment:
This entire block can probably be removed! We shouldn't run installation code in this step. If we add the packages to the dependencies, they may/may not be installed by the `install` step anyway.